adapters/mobkoi/mobkoi_test.go

```diff
@@ -0,0 +1,21 @@
+package mobkoi
```

Comment:
Please add `mobkoi/params_test.go` file to test the valid and invalid bidder params mentioned in the JSON file.